tests/MongoDB.Driver.Tests/Specifications/client-side-encryption/prose-tests/ClientEncryptionProseTests.cs

```diff
@@ -1562,6 +1572,104 @@ DisposableEnvironmentVariable ConfigureDisposableEnvironmentVariable(string key,
                     return new DisposableEnvironmentVariable(key, null); // unset env variable
                 }
             }
+
+            IDisposable ConfigureEnvironment()
```

Comment:
some joy here is still required

Reply:
Given the complexities around GCP configuration, let's move this method into its own helper class to make it more discoverable.

Reply:
I simplified this logic